sdk/src/client/error.rs

```diff
@@ -25,6 +25,14 @@ pub enum Error {
     /// Block dtos error
     #[error("{0}")]
     ApiTypes(#[from] crate::types::api::core::error::Error),
+    /// Invalid bech32 HRP, should match the one from the used network
+    #[error("invalid bech32 hrp for the connected network: {provided}, expected: {expected}")]
+    Bech32HrpMismatch {
+        /// The bech32 human readable part from the provided address.
+        provided: String,
```

Comment:
Should these be Hrps ?

Reply:
I tried that, but I would have to box them and it felt pointless.